src/InertialSense.cpp

```diff
@@ -1237,101 +1237,105 @@ void InertialSense::SaveFlashConfigFile(std::string path, int pHandle)
 
 int InertialSense::LoadFlashConfig(std::string path, int pHandle)
 {
-#if 0
+#if 1
```

Comment:
Remove these (since they are always true, there is no point in them being conditional), or change the 1 to a macro flag (ie, #ifdef USE_YAML_CPP)

Reply:
Done